easybuild/easyconfigs/o/OSU-Micro-Benchmarks/OSU-Micro-Benchmarks-5.7.1-gompi-2021a-CUDA-11.3.1.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'ConfigureMake'
+
+name = 'OSU-Micro-Benchmarks'
+version = '5.7.1'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://mvapich.cse.ohio-state.edu/benchmarks/'
+description = """OSU Micro-Benchmarks"""
+
+toolchain = {'name': 'gompi', 'version': '2021a'}
+
+source_urls = ['https://mvapich.cse.ohio-state.edu/download/mvapich/']
+sources = [SOURCELOWER_TGZ]
+
+dependencies = [
+    ('CUDAcore', '11.3.1', '', True),
+    ('UCX-CUDA', '1.10.0', versionsuffix),
+]
+
+configopts = 'CC="$MPICC" CXX="$MPICC"'
+configopts += ' --enable-cuda --with-cuda=$EBROOTCUDA'
```

Comment:
Shouldn't that be EBROOTCUDACORE?

Reply:
yes (it seems osu micro benchmarks managed to find all the cuda stuff anyway)